part-spec/common/powerSequence.json

```diff
@@ -0,0 +1,137 @@
+                    "examples": [
+                        "vdd",
+                        "en"
+                    ],
+                    "type": "string"
+                },
+                "signal1TerminalIdentifiers": {
+                    "description": "list of component pins associated with signal 1",
+                    "type": "array",
+                    "examples": [
+                        "23",
+                        "A1"
+                    ],
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "signal2": {
+                    "description": "signal that comes up second in the sequence ",
+                    "examples": [
+                        "vdd",
+                        "en"
+                    ],
+                    "type": "string"
+                },
+                "signal2TerminalIdentifiers": {
+                    "description": "list of component pins associated with signal 2",
+                    "type": "array",
+                    "examples": [
+                        "23",
+                        "A1"
+                    ],
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "timeCondition": {
+                    "description": "time between the signal 1 and signal 2 events",
+                    "comment": "units of time",
+                    "$ref": "../common/values.json#/valueOptions"
+                },
+                "powerDirection": {
+                    "description": "whether this is a power up or power down event",
+                    "enum": [
+                        "up",
+                        "down"
+                    ]
+                },
+                "nextPowerState": {
+                    "description": "power state the device enters after the signal transition",
+                    "type": "string",
+                    "examples": [
+                        "suspend",
+                        "active"
+                    ]
+                },
+                "currentPowerState": {
+                    "description": "power state the device is currently in before the signal transition",
+                    "type": "string",
+                    "examples": [
+                        "suspend",
+                        "active"
+                    ]
+                },
+                "riseTime": {
+                    "description": "time for a signal to go from low to high (signal 1 only)",
+                    "comment": "units of time",
+                    "$ref": "../common/values.json#/valueOptions"
+                },
+                "slewRate": {
+                    "description": "maximum rate at which a voltrage rail changes per time (signal 1 only)",
```

Comment:
Typo: "voltage", not "voltrage".
Also. "signal2", not "signal 1"

Reply:
fixed typo. reworded